scripts/tasks/sass.js

```diff
@@ -1,119 +1,50 @@
-module.exports = function(options) {
-  const glob = require('glob');
-  const path = require('path');
-  const requireResolveCwd = require('../require-resolve-cwd');
+// @ts-check
```

Comment:
I'm curious why some tasks (like `rig` and `prettier`) don't have this pragma. Should they?

Reply:
Some of those tasks are actually copy-pasted code from existing tasks. Some of those js files end up having tons warnings because they weren't coded with this pragma. I think these can be corrected in a later fix.